android/app/src/profile/AndroidManifest.xml

```diff
@@ -5,4 +5,6 @@
          to allow setting breakpoints, to provide hot reload, etc.
     -->
     <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.VIBRATE"/>
+    <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED"/>
```

Comment:
Adding in `android/app/src/main/AndroidManifest.xml` is sufficient, no need to add here.